packages/react/src/__tests__/index-test.js

```diff
@@ -50,6 +51,15 @@ describe('Carbon Components React', () => {
         "FormGroup",
         "FormItem",
         "FormLabel",
+        "Header",
```

Comment:
Just realized that `HeaderContainer` (new state manager) is missing, and it's probably my fault. I didn't include `export HeaderContainer from './HeaderContainer';` in `/src/components/UIShell/index.js`. Is that the reason it's not be included with everything else in the `*`?

Reply:
No sweat! I bet that's why. Want to commit the update? I think updating it in the `UISHell/index.js` and then doing:

```
cd packages/react
yarn test -u
```

Will update the snapshot 👍